front/app/components/admin/AddCollaboratorsModal/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+// Components
+import { Box, Button, Text } from '@citizenlab/cl2-component-library';
+import Modal from 'components/UI/Modal';
+import SeatInfo from 'components/SeatInfo';
+
+// Translation
+import { FormattedMessage, useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+// hooks
+import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
+import useSeats from 'api/seats/useSeats';
+
+// Utils
+import { isNil } from 'utils/helperUtils';
+
+interface Props {
+  showModal: boolean;
+  closeModal: () => void;
+  addModerators: () => void;
+  // Total number of seats to add, including admin seats
+  noOfSeatsToAdd: number;
+  // These don't include admin seats
+  noOfSeatsToBuy?: number;
+}
+
+const AddCollaboratorsModal = ({
+  showModal,
+  closeModal,
+  addModerators,
+  noOfSeatsToAdd,
+  noOfSeatsToBuy = noOfSeatsToAdd,
+}: Props) => {
+  const { formatMessage } = useIntl();
+  const { data: appConfiguration } = useAppConfiguration();
+  const { data: seats } = useSeats();
+
+  if (!appConfiguration || !seats) return null;
+
+  const maximumCollaborators =
+    appConfiguration.data.attributes.settings.core.maximum_moderators_number;
+  const currentCollaboratorSeats =
+    seats.data.attributes.project_moderators_number;
+  const hasReachedLimit =
+    !isNil(maximumCollaborators) &&
+    currentCollaboratorSeats >= maximumCollaborators &&
+    noOfSeatsToBuy > 0;
```

Comment:
noOfSeatsToBuy should not be needed to determine if we reached our limit. Current + maximum no of collaborators should be enough, no?

Reply:
This is admin related. If we go with admins, then it is required